espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/Click.kt

```diff
@@ -16,25 +16,14 @@
 
 package io.appium.espressoserver.lib.handlers
 
-import androidx.test.espresso.PerformException
 import io.appium.espressoserver.lib.handlers.exceptions.AppiumException
-import io.appium.espressoserver.lib.handlers.exceptions.InvalidElementStateException
 import io.appium.espressoserver.lib.model.AppiumParams
-import io.appium.espressoserver.lib.model.Element
-
-import androidx.test.espresso.action.ViewActions.click
+import io.appium.espressoserver.EspressoServerRunnerTest.Companion.context
 
 class Click : RequestHandler<AppiumParams, Void?> {
 
     @Throws(AppiumException::class)
     override fun handleInternal(params: AppiumParams): Void? {
-        val viewInteraction = Element.getViewInteractionById(params.elementId)
-        try {
-            viewInteraction.perform(click())
-        } catch (e: PerformException) {
-            throw InvalidElementStateException("click", params.elementId!!, e)
-        }
-
-        return null
+        return context.driverStrategy.click(params)
```

Comment:
we could probably drop curly brackets for this method as it only contains on line

Reply:
done